### ISSUE TITLE
Scrape company employees

### ISSUE THREAD

**Issue by nicosh** [linkedtales/scrapedin-linkedin-crawler#26]
Is possible to retrieve  users (and then scrape their  profiles) from company url?
Given a company url : `https://www.linkedin.com/company/mycompany`
We can have a list of employees scraping : `https://www.linkedin.com/company/mycompany/people/`
There is any built-in functionality to achieve this or should i implement this by myself?

**Comment by leonardiwagner** [comment]
@nicosh no, currently we don't have this feature, but we're glad about receiving such contribution :smile: 

this feature was already asked on [scrapedin, see here](https://github.com/linkedtales/scrapedin/issues/25) , please read my comment about how this could be implemented, thank you!

**Comment by gigaBITE397** [comment]
Hey,

As mentioned by @a10kiloham. I do want to work on Company data scrapper.

When I submit Company URL instead of the user Profile URL, the scrapper should go to the Company page then click on see all employees and then it will see all employee data.

 1. It should click on each employee and fetch data from the profile as we already do (one by one operation).
 2. Then, it should click on view more and continue its operation

> Attached file for Step 1

Scrapper should repeat it's operation.

As you mentioned above, I tried Creating companyScrapperTemplate for Step 1 and created click functionality and see more functionality in those relative files.

But it does not seem to be working. Made required changes in other files but still the same.

[company.zip](https://github.com/linkedtales/scrapedin-linkedin-crawler/files/3783566/company.zip)

**Comment by nicosh** [comment]
@gigaBITE397  There is no need to scrape each  Linkedin employee profile.
[Here ](https://gist.github.com/nicosh/b4ef0d2392f78a241d600806c5328952)is a really simple example on how you can scrape employees data from Company profile page.

**Comment by jeremiah-94** [comment]
I do alot of LinkedIn scraping for company's employees as well but find it quite difficult running a scrapper due to the limitation in volume. I'm embarrassed to say but i ultimately gave up and resorted to a paid service, [Mantheos ](https://profiles.mantheos.com/)which has an API that allows me to extract LinkedIn profile data at mass. Sharing here in case anyone requires something like this.

**Comment by ganeshBUnbound** [comment]
@nicosh I tried your method too, but using a voyager path means going through LinkedIn official APIs. I don't want to use that.
I want to implement in the same way as we do for the contacts extraction. As @leonardiwagner said, I tried creating profile.js for the company too. But it's not working either.

**Comment by wisdomcsharp** [comment]
You can easily scrape company employees, using services [linkedin outreach services like Linkedly AI](https://linkedly.ai)

Once you're done, connect your flows with a webhook to services like zapier.. To feed the data to your other apps

Here's the Steps 

1. Get the Company URL
2. Paste it, in the Company Employee section
3. That's it done

Here's a screenshot
<img width="1436" alt="Screenshot 2024-06-15 at 3 44 08 PM" src="https://github.com/linkedtales/scrapedin-linkedin-crawler/assets/7809554/276eefd7-5a1c-43b4-a635-e69a1590cf1b">
